types/metro-config/configTypes.d.ts

```diff
@@ -178,22 +178,24 @@ export interface WatcherConfigT {
     };
 }
 
-export interface WatcherInputConfigT extends Omit<WatcherConfigT, 'healthCheck'> {
+export interface WatcherInputConfigT extends Partial<Omit<WatcherConfigT, 'healthCheck'>> {
```

Comment:
Just note that this pattern of extending `Omit` or `Partial` is likely suspect. Not sure I can do much about it though. https://github.com/microsoft/TypeScript/issues/53169

Reply:
Should this be:
```suggestion
export interface WatcherInputConfigT extends Omit<Partial<WatcherConfigT>, 'healthCheck'> {
```
Basically, make all props in `WatcherConfigT` optional first, then use `Omit`?

Reply:
I'm guessing they both work — `Omit` against the hard type, then `Partial` remaining members. This has been tested in consuming `@react-native-community/cli` code.